package.json

```diff
@@ -109,7 +109,7 @@
   },
   "husky": {
     "hooks": {
-      "pre-commit": "rm -rf docs/api && typedoc --plugin typedoc-plugin-markdown --hideBreadcrumbs true && git add -A docs/api/*.md"
+      "pre-commit": "rm -rf docs/api && typedoc --plugin typedoc-plugin-markdown --hideBreadcrumbs true && git add -A docs/api/"
```

Comment:
No, there are only .md. I removed extension because if there are only .md files in this folder, then there is no need to specify file type, but if there are other types of file can be there, then they should also be included in commit